### DISCUSSION TITLE
ControlNet web api access?

### DISCUSSION THREAD

**Discussion by joeydee2025** [Mikubill/sd-webui-controlnet#97] [General]
Working great so far :)
Does anybody know, do I have any web api access for ControlNet options? I'm using the automatic1111 web api as described in [localserver]/docs with my c# application successfully, 
I'd like to set the ControlNet enable flag, models, input image etc. via post methods before calling my txt2img or img2img post request in my c# app.
Possible? Any hints or documentation?

**Comment by Ryu1845** [comment]
Related: #63
